### PR TITLE
Replace uses of severityColorMap from severityColors with accessible alternatives

### PR DESCRIPTION
**Pull request by pedrottimark**
### Problems

1. Color text causes accessibility problems like assumption of perfect color vision.

2. Inconsistent colors for severity throughout UI.

### Analysis

1. src/Containers/ConfigManagement/Dashboard/widgets/PolicyViolationsBySeverity.js

    * This use of `Sunburst` element is similar to `CvesByCvssScore` component.

2. src/Containers/VulnMgmt/widgets/TopRiskyEntitiesByVulnerabilities.js

    * This is the only use of `Scatterplot` element.

### Solution

1. Except for **policy** colors, labels, and severities, instead of vulnerability, changes are similar to src/Containers/VulnMgmt/widgets/CvesByCvssScore.js in #5811

2. Encapsulate `legendData` within component.

## Checklist
- [x] Investigated and inspected CI test results
- [x] ~Unit test and regression tests added~

### Integration testing

In ui/apps/platform folder:

1. `export CYPRESS_ROX_POSTGRES_DATASTORE=true`

2. `yarn cypress-open`

    * configmanagement/dashboard.test.js
    * vulnmanagement/dashboard.test.js
    * vulnmanagement/dashboardToEntityPage.test.js

### Manual testing

1. Visit /main/configmanagement and see widget at upper left

    * classic severity colors in dark theme
        Sunburst_dark_classic

    * PatternFly severity colors in dark theme
        Sunburst_dark_PatternFly

    * classic severity colors in light theme
        Sunburst_light_classic

    * PatternFly severity colors in light theme
        Sunburst_light_PatternFly

2. Visit /main/vulnerability-management and see widget at upper left

    * classic severity colors in dark theme
        Scatterplot_dark_classic

    * PatternFly severity colors in dark theme
        Scatterplot_dark_PatternFly

    * classic severity colors in light theme
        Scatterplot_light_classic

    * PatternFly severity colors in light theme
        Scatterplot_light_PatternFly

### CODE REVIEW

File: ui/apps/platform/src/Containers/VulnMgmt/widgets/TopRiskyEntitiesByVulnerabilities.js [hunk 1]
```diff
@@ -14,19 +14,24 @@ import HoverHintListItem from 'Components/visuals/HoverHintListItem';
...
 import { checkForPermissionErrorMessage } from 'utils/permissionUtils';
 import { getSeverityByCvss } from 'utils/vulnerabilityUtils';
 import { entitySortFieldsMap, cveSortFields } from 'constants/sortFields';
 import { WIDGET_PAGINATION_START_OFFSET } from 'constants/workflowPages.constants';
 import { entityPriorityField } from 'Containers/VulnMgmt/VulnMgmt.constants';
 import useFeatureFlags from 'hooks/useFeatureFlags';
 
+// Beware, policy instead of vulnerability severities because of getSeverityByCvss function!
+
+const policySeverities = ['LOW_SEVERITY', 'MEDIUM_SEVERITY', 'HIGH_SEVERITY', 'CRITICAL_SEVERITY'];
```
Comment (dvail) [inline ui/apps/platform/src/Containers/VulnMgmt/widgets/TopRiskyEntitiesByVulnerabilities.js:28]: Nit: what about importing this from `policy.proto.ts`? (Ditto on the above file)

Comment (pedrottimark) [reply ui/apps/platform/src/Containers/VulnMgmt/widgets/TopRiskyEntitiesByVulnerabilities.js:28]: Yes, indeed. I forgot that the opposite order from Critical to Low in legend of `PolicyViolationsBySeverity` is independent of the order of the severity in the arrays.

File: ui/apps/platform/src/Containers/ConfigManagement/Dashboard/widgets/PolicyViolationsBySeverity.js [hunk 2]
```diff
@@ -10,18 +10,23 @@ import { Link, withRouter } from 'react-router-dom';
...
 
-const passingLinkColor = 'var(--base-500)';
+const policySeverities = ['LOW_SEVERITY', 'MEDIUM_SEVERITY', 'HIGH_SEVERITY', 'CRITICAL_SEVERITY'];
+
+const legendData = policySeverities.map((severity) => ({
+    title: policySeverityLabels[severity],
+    color: policySeverityColorMap[severity],
+}));
+
+const linkColor = 'var(--primary-600)'; // TODO might be able to remove after PatternFly dark them
```
Comment (vjwilson) [inline ui/apps/platform/src/Containers/ConfigManagement/Dashboard/widgets/PolicyViolationsBySeverity.js:28]: nit: spelling of "theme"
```suggestion
const linkColor = 'var(--primary-600)'; // TODO might be able to remove after PatternFly dark theme
```

**Review by vjwilson — APPROVED** [review]

File: ui/apps/platform/src/Containers/ConfigManagement/Dashboard/widgets/PolicyViolationsBySeverity.js (hunk 2 above)
Comment (pedrottimark) [reply ui/apps/platform/src/Containers/ConfigManagement/Dashboard/widgets/PolicyViolationsBySeverity.js:28]: Good eyes! Unintentional “othering” of theme.

**Pull request merged**